packages/react/src/generators/application/lib/add-e2e.ts

```diff
@@ -52,6 +52,15 @@ export async function addE2e(
         baseUrl: 'http://localhost:4200',
         jsx: true,
         rootProject: options.rootProject,
+        webServerCommands: hasNxBuildPlugin
+          ? {
+              default: `nx run ${options.projectName}:serve`,
+              production: `nx run ${options.projectName}:serve:production`,
```

Comment:
Production configuration may not be there since the "production" server may be through `preview` target.